src/js/newTab.js

```diff
@@ -69,7 +101,7 @@
       }
     },
 
-    setInitialState(message) {
+    _setInitialState(message) {
```

Comment:
I couldn't see any good reason for this to be public, so I switched it to private.